_includes/exhibits.js

```diff
@@ -39,19 +37,6 @@ const decode = function(txt) {
   }
 };
 
-const arrayEqual = function(a, b) {
-  if (a.length != b.length) {
-    return false;
-  }
-  var pairs = [];
-  for (var i = 0; i < a.length; i++) {
-    pairs.push([a[i], b[i]]);
-  }
-  return pairs.every(function(p) {
-    return p[0] == p[1];
-  });
-};
-
```

Comment:
Old function for comparing arrays. Last used to set an old change tracking flag.